reporting/api/src/main/java/org/jboss/windup/reporting/config/classification/Classification.java

```diff
@@ -49,20 +51,10 @@
     private List<Link> links = new ArrayList<>();
     private Set<String> tags = new HashSet<>();
     private List<Quickfix> quickfixes = new ArrayList<>();
-
-    /**
-     * @return the quickfixes
-     */
-    public List<Quickfix> getQuickfixes()
-    {
-        return quickfixes;
-    }
-
     private RegexParameterizedPatternParser classificationPattern;
     private RegexParameterizedPatternParser descriptionPattern;
     private int effort;
-    private Severity severity = EffortReportModel.DEFAULT_SEVERITY;
-
+    private IssueCategory issueCategory = null;
```

Comment:
It is still setting the category to a default, it is just doing it farther down in the code now.